server/src/api/api.graphql

```diff
@@ -3,6 +3,7 @@ type Query {
 	user: User
 	item(id: Int!): Item
 	items: [Item!]
+    categories: [Category!]
```

Comment:
A category for the user?

Reply:
DB makes it seem not so, maybe edit the comment

Reply:
It's item category, will add comment